### ISSUE TITLE
How to use this browser to remote proctoring students with WebRTC video?

### ISSUE THREAD

**Issue by fishfree** [SafeExamBrowser/Chrominimum#2]
Maybe this is only a problem of documentaton?

**Comment by dbuechel** [comment]
Install SEB for Windows and Chrominimum and then configure Chrominimum as third-party application, see https://safeexambrowser.org/windows/win_usermanual_en.html#PermittedProcessesSection

**Comment by fishfree** [comment]
@dbuechel Thank you! But may I reopen this issue? I tried your tip, but no luck. I specified the start URL of Chrominimum as the quiz URL of Moodle as the screenshot below:
![image](https://user-images.githubusercontent.com/1741341/89754290-e9b17380-db0d-11ea-850c-d5abbe667c03.png)
But when start SEB 3.0, it shows as the photo below:
![image](https://user-images.githubusercontent.com/1741341/89754317-0188f780-db0e-11ea-8ade-48eff94f1ce6.png)
